### PR TITLE
Modify host initiators listing page

### PR DESCRIPTION
**Pull request by llivne**
two fields were added to host initiators listing page:
- status
- host_cluster_name

Screenshot 2021-07-28 150036

links
-----
https://github.com/ManageIQ/manageiq-providers-autosde/pull/73 - need to be merged after this PR
https://github.com/ManageIQ/manageiq-ui-classic/pull/7803 - need to be merged after this PR

### CODE REVIEW

File: spec/support/table_list.txt [hunk 1]
```diff
@@ -332,6 +331,7 @@ users
 vim_performance_operating_ranges
 vim_performance_states
 vms
+volume_mappings
```
Comment (agrare) [inline spec/support/table_list.txt:334]: This looks unrelated, can you remove from this commit?

Comment (llivne) [reply spec/support/table_list.txt]: done

**Review by agrare — APPROVED** [review]
